### DISCUSSION TITLE
member login

### DISCUSSION THREAD

**Discussion by dendynugroho** [hotspotbilling/phpnuxbill#336] [Q&A]
Mas @ibnux dan teman teman yang lain mohon pencerahan nya.

pada saat member/user/customer login tidak bisa, padahal user sudah ditambahkan.

https://github.com/user-attachments/assets/9176056a-590b-44a9-9892-d333f727357c

**Comment by hasalioma3** [comment]
hi @dendynugroho did you get a solution for this?
I am also facing the same issue
